perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -4131,6 +4141,7 @@ public boolean removeAttributeWithoutCheck(PerunSession sess, Resource resource,
 			throw new InternalErrorException(ex);
 		}
 		if (changed)
+			log.info("{} removed attribute({}) from esource. Id resource = {}",sess.getPerunClient().getId(), attribute.getName(), resource.getId());
```

Comment:
``esource`` -> ``resource``
Also you probably don't want () around {} which is replaced for attr name, since in other messages you don't do that.